ci/unit/test_main_node.sh

```diff
@@ -48,9 +48,28 @@ if ${NLT:-false}; then
     mkdir -p vm_test
     ./utils/node_local_test.py --output-file=vm_test/nlt-errors.json all
 else
-    IS_CI=true OLD_CI=false RUN_TEST_VALGRIND="$WITH_VALGRIND" utils/run_test.sh
-
+    ls
+    ls test_results || true
+    ls test_results_debug || true
+    IS_CI=true OLD_CI=false RUN_TEST_VALGRIND="$WITH_VALGRIND" RUN_TEST_FILTER="vos_test" utils/run_test.sh
+    ls
+    ls test_results_debug || true
     if [ "$WITH_VALGRIND" == 'memcheck' ]; then
-	mv test_results/unit-test-*.memcheck.xml .
+	mv test_results_debug/unit-test-*.memcheck.xml .
+	# Debugging
+	ls unit-test-*.memcheck.xml
+	for i in $(ls unit-test-*.memcheck.xml); do
+	    kind="$(grep "<kind>" $i || true)"
+	    if [ ! -z "$kind" ]; then
+	        echo ">> $i";
+	        echo $kind;
+		echo ""
+	    fi;
+	done
+        echo "marj>> xmls content"
+        for i in $(ls unit-test-*.memcheck.xml); do
+            echo ">> file is $i"
+            cat $i 
```

Comment:
(style)  trailing whitespace
(lint) Double quote to prevent globbing and word splitting. [SC2086]